### ISSUE TITLE
When targeted UFOs return to the alien dimension, friendly ships switch targets to attack random point on map

### ISSUE THREAD

**Issue by dethstryke01** [OpenApoc/OpenApoc#1325]
While attacking an enemy ship, my hoverbikes decided to switch targets onto a random point on the map. This appears to have occurred exactly when I clicked on the "organizations" tab with the map open.

https://youtu.be/BRogQm5x4Iw?t=1146

The random point on the map also had a red "enemy" box, but no enemies listed on the enemy ships tab were at that location. 

https://youtu.be/BRogQm5x4Iw?t=1207

Window 10 x64 22H2
OpenApoc version 0.0.930
no mods in use

[settings.zip](https://github.com/OpenApoc/OpenApoc/files/14152382/settings.zip)

[save_test 3.zip](https://github.com/OpenApoc/OpenApoc/files/14152385/save_test.3.zip)

Here are two save files where the bug is happening, and more video: 

https://youtu.be/-029oEE0ynE?t=3916
https://youtu.be/-029oEE0ynE?t=4041

[save_ships attack random spot.zip](https://github.com/OpenApoc/OpenApoc/files/14237787/save_ships.attack.random.spot.zip)
[save_ships attack random spot 2.zip](https://github.com/OpenApoc/OpenApoc/files/14237791/save_ships.attack.random.spot.2.zip)

**Comment by Kurtsley** [comment]
I don't think it was the click on the organizations tab. Your ships continue to try to attack the ufo even after it has warped back to the alien dimension. I thought this was fixed already...

**Comment by dethstryke01** [comment]
I'm not sure exactly what's going on. When I watched it in slow motion it looked like the click on the organizations tab has something to do with it, but it could be a coincidence.

**Comment by dethstryke01** [comment]
Here's another similar situation where friendly ships switched targets onto a random map point. It does look like the problem began after their target disappeared. 

https://youtu.be/ou0NPvZk_x8?t=141

[settings.zip](https://github.com/OpenApoc/OpenApoc/files/14178771/settings.zip)
[save_test 9.zip](https://github.com/OpenApoc/OpenApoc/files/14178775/save_test.9.zip)

**Comment by dethstryke01** [comment]
Here are two save files where the bug is happening, and more video: 

https://youtu.be/-029oEE0ynE?t=3916
https://youtu.be/-029oEE0ynE?t=4041

[save_ships attack random spot.zip](https://github.com/OpenApoc/OpenApoc/files/14237787/save_ships.attack.random.spot.zip)
[save_ships attack random spot 2.zip](https://github.com/OpenApoc/OpenApoc/files/14237791/save_ships.attack.random.spot.2.zip)

**Comment by BeornTB** [comment]
> I don't think it was the click on the organizations tab. Your ships continue to try to attack the ufo even after it has warped back to the alien dimension. I thought this was fixed already...

It is a known bug and I don't think a fix was implemented
